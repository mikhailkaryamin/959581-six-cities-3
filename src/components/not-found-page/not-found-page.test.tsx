import * as React from 'react';
import * as renderer from 'react-test-renderer';
import NotFoundPage from './not-found-page';

test(`Not found page render correctly`, () => {
  const tree = renderer
    .create(
        <NotFoundPage />
    ).toJSON();
  expect(tree).toMatchSnapshot();
});
