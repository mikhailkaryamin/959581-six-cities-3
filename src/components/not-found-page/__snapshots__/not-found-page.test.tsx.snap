// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Not found page render correctly 1`] = `
<main
  className="page__main page__main--index page__main--index-empty"
>
  <div
    className="cities"
  >
    <div
      className="cities__places-container cities__places-container--empty container"
    >
      <section
        className="cities__no-places"
      >
        <div
          className="cities__status-wrapper tabs__content"
        >
          <b
            className="cities__status"
          >
            Page not found
          </b>
          <p
            className="cities__status-description"
          >
            Maybe you were wrong when entering link
          </p>
        </div>
      </section>
      <div
        className="cities__right-section"
      />
    </div>
  </div>
</main>
`;
